.github/workflows/error-codes-updater.yaml

```diff
@@ -3,6 +3,9 @@ on:
   push:
     branches:
       - "master"
+  pull_request:
+    branches:
+      - "master"
```

Comment:
This is a good change and we can merge this change in a separate PR immediately.

The pre-commit hook needs to be qualified in terms of avoiding duplicate runs of `make error`. `make server` does not include the `error` target, which is good in so much as avoiding duplicate runs (slowing DevEx). Is there anywhere else that `error` is run?

Reply:
In mesheryctl too we have make error target
So should i create a new one for workflow

Reply:
ok, sounds good.